man/get_source_expressions.Rd

```diff
@@ -52,3 +52,13 @@ newline (as determined by \code{\link[=readLines]{readLines()}} producing a corr
 \description{
 This object is given as input to each linter
 }
+\details{
+The file is read in using the \code{encoding} setting.
+This setting found by taking the first valid result from the following locations
+\enumerate{
+\item The \code{encoding} key from the usual lintr configuration settings.
```

Comment:
link to `?default_settings` here

Reply:
`?default_settings` needs a little love I think. Currently it contains not much more than "An object of class list of length 12".
Maybe that would be a good place in general to document settings?

Reply:
I picked `?default_settings` kind of at random, more that "from the usual lintr configuration settings" won't be helpful if this is the first help page a user sees -- some link to help about what we mean there is needed.

If we don't have such a page yet then yes, let's use `?default_settings` :)